README.md

```diff
@@ -1,19 +1,53 @@
-Absolute
-=============
-
+# Absolute
 [![Linux/Max Build Status](https://travis-ci.org/lunchclass/absolute.svg?branch=master)](https://travis-ci.org/lunchclass/absolute/branches)
-[![Windows Build status](https://ci.appveyor.com/api/projects/status/099u4iekeny4lpsa/branch/master?svg=true)](https://ci.appveyor.com/project/romandev/absolute/branch/master)
+[![Windows Build status](https://ci.appveyor.com/api/projects/status/pmac6fwbclavu8n0/branch/master?svg=true)](https://ci.appveyor.com/project/romandev/absolute/branch/master)
```

Comment:
Why do we change this link?

Reply:
I didn't know that the build state address of my repo was different from the build state address of origin. I'll fix it soon